lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -260,6 +261,10 @@ def document_download_link_if_opposite_sex_resident_of_uk_countries?
       MarriageAbroadDataQuery::NO_DOCUMENT_DOWNLOAD_LINK_IF_OS_RESIDENT_OF_UK_COUNTRIES.exclude?(ceremony_country)
     end
 
+    def consular_fee(service)
+      @rates_query.rates[service]
```

Comment:
Maybe. I guess I'm used to validation occurring before we call render, and that render outputs what data is available and can deal with missing values in a safe way.